lib/cooked_post_processor.rb

```diff
@@ -276,7 +281,8 @@ def add_lightbox!(img, original_width, original_height, upload, cropped: false)
     lightbox.add_child(img)
 
     # then, the link to our larger image
-    src = UrlHelper.cook_url(img["src"], secure: @post.with_secure_uploads?)
+    src_url = img["src"].include?("secure-uploads") ? upload&.url : img["src"]
```

Comment:
Is `img[“src”]` at any point a user provided value? If it is, could people add “secure-upload” is the src (as a query string for example)? Would that cause any problem? 🤔

Reply:
Hmm yeah that is a good point...let me do something slightly smarter here